shell/client/shell.html

```diff
@@ -1094,7 +1094,7 @@ <h4 class="share-with-others">Share with others</h4>
       </span>
 
       <label>Relay port:
-        <input type="text" name="smtpPort" value="{{smtp.port}}" />
+        <input type="text" name="smtpPort" value="{{smtp.port}}" pattern="[0-9]+" />
```

Comment:
Maybe use `type="range" min="1" max="65535"`? This way the browser can give a better error when the value doesn't match (and I think will even display a numeric keypad on mobile). Alternatively, give the input a `title` which will be displayed on error.